core/src/main/java/com/klaytn/caver/wallet/keyring/SignatureData.java

```diff
@@ -160,8 +166,17 @@ public RlpList toRlpList() {
         byte[] r = Numeric.hexStringToByteArray(getR());
         byte[] s = Numeric.hexStringToByteArray(getS());
 
+        RlpString rlpV;
+        byte[] trimmedV = Bytes.trimLeadingZeroes(v);
+        if (trimmedV[0] == 0) {
```

Comment:
Why this code is needed? Would you explain to me?

Reply:
Sure.
New signer can have v value(called y parity) `1` or `0`.
If given v is `0` we must treat it as integer 0 (will be encoded as 0x80).

Current source code without that lines
handles encode `0` as `0x00`.

I left comment at the line 172.
`// When given v value is "0x0", it must be treated as integer 0 by encoding rule.`